src/routes/semrush-route.php

```diff
@@ -95,4 +150,24 @@ public function authenticate( WP_REST_Request $request ) {
 	public function has_valid_code( $code ) {
 		return $code !== '';
 	}
+
+	/**
+	 * Checks if a valid country code was submitted.
+	 *
+	 * @param string $country_code The country code to check.
+	 *
+	 * @return boolean Whether or not the country code is valid.
+	 */
+	public function has_valid_country_code( $country_code ) {
+		return ( $country_code !== '' && preg_match( '/^[a-z]{2}$/', $country_code ) === 1 );
```

Comment:
In the SEMrush documentation https://www.semrush.com/api-analytics/#databases, I see some country codes are prepended with `mobile-`. For example: `mobile-us`, ``mobile-nl`, `mobile-it`. I have no idea if we're going to use these "mobile" databases but if so, this regex won't work as expected.

Reply:
The `mobile-` ones have been excluded by Product.
I agree that we shoud "centralize" the country code list (at the moment is hardcoded in the component).